deliveries/bagpipes_m2.md

```diff
@@ -24,7 +24,7 @@ Bagpipes USDC (Ethereum): 0x13d3a4dc4eddc79e5f277d4633834ba6ada15a7f
 | Number | Deliverable | Specification |
 | -----: | ----------- | ------------- |
 | 0. | Documentation | Public documentation [https://docs.bagpipes.io/docs/nodes/chainTx](https://docs.bagpipes.io/docs/nodes/chainTx)  | |
-| 1. | automationPrice and automationTime pallet integration | **Overdelivery, we support every pallet in Turing and Oak. This was really the only way to do it, because the pallets were much more complex than initially imagined, with depply nesting structures, particularly used with XCM.** | 
+| 1. | automationPrice and automationTime pallet integration | **Overdelivery, we support every pallet in Turing and Oak. This was really the only way to do it, because the pallets were much more complex than initially imagined, with depply nesting structures, particularly used with XCM.** here is bagpipe template for [automationTime](https://alpha.bagpipes.io/#/create/?diagramData=XEtWDwnDY) in a chainTx node.   | 
```

Comment:
The claim of “over-delivered” doesn’t seem accurate. We need to stay aligned with the goal of the grant, which is to trigger a swap transaction on Mangata using the automation pallet extrinsics. The claim that “we support every pallet in Turing” wasn’t part of that goal. While the `schedule_xcm_task `function in your template is correct, the goal is to abstract it, rather than requiring users to manually fill out every field.

I personally couldn’t fill them all out without additional programmatic assistance. Without proper abstraction, it’s difficult for users to interact with these extrinsics effectively. Since the primary goal of your product is to streamline the user experience, this template link doesn’t achieve that.

An alternative approach would be to provide a template link with pre-filled values, so the user only needs to adjust the amount and click submit. Unfortunately, this template link doesn’t meet that expectation either.

Reply:
For the reasons mentioned above, a detailed demo walkthrough page is required in the 0. | Documentation section. The current documentation at https://docs.bagpipes.io/docs/nodes/chainTx is insufficient on its own.

Reply:
@chrisli30 you can read the AutomationTime docs are here: 
https://docs.bagpipes.io/docs/scheduletransfer

Try it out here: 
https://app.xcmsend.com/#/create/?diagramData=1Ti7jn5lM 

Its much easier to use using the custom actions, documented in the link above